bpf/lib/process.h

```diff
@@ -274,6 +274,7 @@ struct msg_k8s {
 
 struct heap_exe {
 	char buf[BINARY_PATH_MAX_LEN];
+	char end[BINARY_PATH_MAX_LEN];
```

Comment:
Sorry just noticed that the postfix LPM TRIE maps only go up to STRING_POSTFIX_MAX_LENGTH in size (= 128).
So here should probably be:
```
char end[STRING_POSTFIX_MAX_LENGTH];
```

Reply:
Yes, thank you, that's a good point. I'm fighting with verifier now).

Reply:
Ah sorry. Feel free to paste problem here; we might be able to help.

Reply:
Thanks, I'm think I managed to deal with verifier:)